scripts/installer/install.sh

```diff
@@ -253,7 +253,19 @@ create_bisdn_linux_msdos_partition()
 
 platform_check()
 {
-    /bin/true
+    onie_platform=$(onie-sysinfo -p)
+
+    if [ ! -d "./machine/${onie_platform}" ]; then
+      echo "ERROR: Unknown or unsupported platform: $onie_platform" >&2
+      echo "This image only supports the following platforms:" >&2
+      platforms="$(ls -1 ./machine)"
+      for plat in $flatforms; do
+         echo "    $platform"
```

Comment:
If the earlier lines go to stderr, this line should, too.

Reply:
Also fixed, as well as the use of the correct variable here.